httpbakery/client.go

```diff
@@ -137,9 +137,8 @@ type Client struct {
 // discharge-acquisition process used by a Client.
 type DischargeAcquirer interface {
 	// AcquireDischarge should return a discharge macaroon for the given third
-	// party caveat. The firstPartyLocation holds the location of the original
-	// macaroon.
-	AcquireDischarge(firstPartyLocation string, cav macaroon.Caveat) (*macaroon.Macaroon, error)
+	// party caveat.
+	AcquireDischarge(cav macaroon.Caveat) (*macaroon.Macaroon, error)
```

Comment:
I think that belongs as a code review comment, not as a comment in the code, as there's no particular reason (apart from my misguided reasoning) why we should include the first party location anywhere at all.